### DISCUSSION TITLE
External driver requirements for stlink-1.7.0-x86_64-w64-mingw32?

### DISCUSSION THREAD

**Discussion by patzf** [stlink-org/stlink#1347] [Q&A (resolved)] [answered]
Two questions I could not find a documented answer to:

1.) On Windows 10, 64bit OS and STlink V3 Mini:
Is there any external driver requirement or should it be possible to run stlink-1.7.0-x86_64-w64-mingw32 binaries (st-flash, st-info) without any other drivers (such as the official STLink driver from ST)? One some Windows PCs I get a "libsub not found error" and browsing thourgh the issues I found a lot of bug reports that seem related but all marked as cloed or fixed. I cannot see any (obvious) differents in the machines and why it works on one PC and not the other beside of the fact that one of them has/had officiela STLink drivers installed once.

2.) Where can I get prebuilt binaries for stlink newer than stlink-1.7.0 for Windows 10, 64 bit?

**Comment by Nightwalker-87** [comment, marked as answer]
1) There are no additional driver requirements (apart from the specific hardware used). The windows binary includes and installs the necessary libusb library automatically. Every published release is pre-tested on Windows and Linux. However already existing usb libraries and or drivers or their leftovers may cause issues. This is also the reason why we can not do much about this specific issue, unless asking affected users to ensure a clean install.
2) No, there are no newer pre-compiled binaries until the next release, for which there is no planned release date yet.

**Comment by Nightwalker-87** [comment]
Closes #1350.
